lint failing for package project after fresh npm install

## What you are doing?

Trying to build project to contribute a pull request.

Using head of master b00fc76a52994d4a0e98b04df91f0d802f99619e

```bash
npm install
npm run lint
```

## What do you expect to happen?

Tests to pass

## What is actually happening?

Failing on multiple lint errors.  I've checked i don't have a personal .eslintrc and i also tried pinning to the exact eslint and eslint plugin versions stated as lower bounds, but didn't help...

If i skip lint and run `npm run test-raw` the actual tests pass.

I don't dev on npm packages much but i guess because there's no lockfile the previously passing build could have been using pretty much any specific dependency versions?  

I'm happy to fix these but, as behaviour has changed without intent, thought i better check if this is what you really want?

_Output_

```
> sequelize-cli@6.2.0 lint /home/dmoore/projects/nhs/dependencies/cli
> eslint test src


/home/dmoore/projects/nhs/dependencies/cli/src/commands/seed.js
  47:15  error  Insert `⏎········`  prettier/prettier
  48:9   error  Insert `··`         prettier/prettier
  49:1   error  Insert `··`         prettier/prettier

/home/dmoore/projects/nhs/dependencies/cli/src/core/migrator.js
  69:25  error  Insert `⏎···`  prettier/prettier

/home/dmoore/projects/nhs/dependencies/cli/test/support/helpers.js
  30:12  error  Replace `'rm·-rf·*·&·rm·-rf·.sequelizerc',·{·cwd:·file.path·},·function·(⏎········err⏎······` with `⏎········'rm·-rf·*·&·rm·-rf·.sequelizerc',⏎········{·cwd:·file.path·},⏎········function·(err`                    prettier/prettier
  33:1   error  Insert `··`                                                                                                                                                                                                          prettier/prettier
  34:7   error  Replace `}` with `··}⏎······`                                                                                                                                                                                        prettier/prettier
  56:12  error  Replace `command,·{·cwd:·file.path,·env:·env·},·function·(⏎········err,⏎········stdout,⏎········stderr⏎······` with `⏎········command,⏎········{·cwd:·file.path,·env:·env·},⏎········function·(err,·stdout,·stderr`  prettier/prettier
  61:1   error  Insert `··`                                                                                                                                                                                                          prettier/prettier
  63:1   error  Insert `··`                                                                                                                                                                                                          prettier/prettier
  65:9   error  Insert `··`                                                                                                                                                                                                          prettier/prettier
  66:1   error  Replace `··········` with `············`                                                                                                                                                                             prettier/prettier
  67:9   error  Insert `··`                                                                                                                                                                                                          prettier/prettier
  69:1   error  Insert `··`                                                                                                                                                                                                          prettier/prettier
  70:11  error  Insert `··`                                                                                                                                                                                                          prettier/prettier
  71:1   error  Insert `··`                                                                                                                                                                                                          prettier/prettier
  72:1   error  Replace `··········` with `············`                                                                                                                                                                             prettier/prettier
  73:1   error  Insert `··`                                                                                                                                                                                                          prettier/prettier
  75:9   error  Insert `··`                                                                                                                                                                                                          prettier/prettier
  76:1   error  Replace `··········` with `············`                                                                                                                                                                             prettier/prettier
  77:1   error  Insert `··`                                                                                                                                                                                                          prettier/prettier
  78:1   error  Insert `··`                                                                                                                                                                                                          prettier/prettier
  79:13  error  Insert `··`                                                                                                                                                                                                          prettier/prettier
  80:11  error  Insert `··`                                                                                                                                                                                                          prettier/prettier
  81:1   error  Replace `············` with `··············`                                                                                                                                                                         prettier/prettier
  82:1   error  Insert `··`                                                                                                                                                                                                          prettier/prettier
  83:1   error  Insert `··`                                                                                                                                                                                                          prettier/prettier
  84:13  error  Insert `··`                                                                                                                                                                                                          prettier/prettier
  85:11  error  Insert `··`                                                                                                                                                                                                          prettier/prettier
  86:1   error  Insert `··`                                                                                                                                                                                                          prettier/prettier
  87:11  error  Insert `··`                                                                                                                                                                                                          prettier/prettier
  88:11  error  Insert `··`                                                                                                                                                                                                          prettier/prettier
  89:1   error  Replace `········` with `··········`                                                                                                                                                                                 prettier/prettier
  90:7   error  Replace `}` with `··}⏎······`                                                                                                                                                                                        prettier/prettier

✖ 34 problems (34 errors, 0 warnings)
  34 errors and 0 warnings potentially fixable with the `--fix` option.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! sequelize-cli@6.2.0 lint: `eslint test src`
npm ERR! Exit status 1
```

--- a/src/commands/seed.js
+++ b/src/commands/seed.js
@@ -44,9 +44,10 @@ function seedAll(args) {
 function seedUndoAll(args) {
   return getMigrator('seeder', args)
     .then((migrator) => {
-      return (helpers.umzug.getStorage('seeder') === 'none'
-        ? migrator.pending()
-        : migrator.executed()
+      return (
+        helpers.umzug.getStorage('seeder') === 'none'
+          ? migrator.pending()
+          : migrator.executed()
       ).then((seeders) => {
         if (seeders.length === 0) {
           helpers.view.log('No seeders found.');

--- a/src/core/migrator.js
+++ b/src/core/migrator.js
@@ -66,7 +66,8 @@ export async function getMigrator(type, args) {
 }
 
 export function ensureCurrentMetaSchema(migrator) {
-  const queryInterface = migrator.options.storageOptions.sequelize.getQueryInterface();
+  const queryInterface =
+    migrator.options.storageOptions.sequelize.getQueryInterface();
   const tableName = migrator.options.storageOptions.tableName;
   const columnName = migrator.options.storageOptions.columnName;
 

--- a/test/support/helpers.js
+++ b/test/support/helpers.js
@@ -27,11 +27,13 @@ module.exports = {
 
   clearDirectory: function () {
     return through.obj(function (file, encoding, callback) {
-      exec('rm -rf * & rm -rf .sequelizerc', { cwd: file.path }, function (
-        err
-      ) {
-        callback(err, file);
-      });
+      exec(
+        'rm -rf * & rm -rf .sequelizerc',
+        { cwd: file.path },
+        function (err) {
+          callback(err, file);
+        }
+      );
     });
   },
 
@@ -53,41 +55,41 @@ module.exports = {
 
       logToFile(command);
 
-      exec(command, { cwd: file.path, env: env }, function (
-        err,
-        stdout,
-        stderr
-      ) {
-        var result = file;
+      exec(
+        command,
+        { cwd: file.path, env: env },
+        function (err, stdout, stderr) {
+          var result = file;
 
-        logToFile({ err: err, stdout: stdout, stderr: stderr });
+          logToFile({ err: err, stdout: stdout, stderr: stderr });
 
-        if (stdout) {
-          expect(stdout).to.not.contain('EventEmitter');
-        }
-
-        if (options.pipeStdout) {
-          result = stdout;
-        } else if (options.pipeStderr) {
-          result = stderr;
-        }
-
-        if (options.exitCode) {
-          try {
-            expect(err).to.be.ok();
-            expect(err.code).to.equal(1);
-            callback(null, result);
-          } catch (e) {
-            callback(
-              new Error('Expected cli to exit with a non-zero code'),
-              null
-            );
+          if (stdout) {
+            expect(stdout).to.not.contain('EventEmitter');
           }
-        } else {
-          err = options.pipeStderr ? null : err;
-          callback(err, result);
+
+          if (options.pipeStdout) {
+            result = stdout;
+          } else if (options.pipeStderr) {
+            result = stderr;
+          }
+
+          if (options.exitCode) {
+            try {
+              expect(err).to.be.ok();
+              expect(err.code).to.equal(1);
+              callback(null, result);
+            } catch (e) {
+              callback(
+                new Error('Expected cli to exit with a non-zero code'),
+                null
+              );
+            }
+          } else {
+            err = options.pipeStderr ? null : err;
+            callback(err, result);
+          }
         }
-      });
+      );
     });
   },
 

